build.gradle

```diff
@@ -1,97 +1,118 @@
-apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
-apply plugin: 'application'
 apply plugin: 'jacoco'
-apply from: 'http://dl.bintray.com/shemnon/javafx-gradle/8.1.1/javafx.plugin'
 
-mainClassName = "edu.wpi.grip.Main"
-version = '0.1.0'
+allprojects {
```

Comment:
Is there a reason to reorder these statements?  If not, you should revert the order